packages/open-ui/src/interfaces/BorderRadiusOption.ts

```diff
@@ -0,0 +1 @@
+export type BorderRadiusOption = 'small' | 'medium' | 'large';
```

Comment:
@remotealex we are exporting Types but naming them as interfaces :thinking: ?

Reply:
Yeah, i think thats fine. They're basically the same :)

Reply:
fair then :)